lib/system/redis_pool.rb

```diff
@@ -9,7 +9,11 @@ class RedisPool
     def initialize(config={})
       config = config.dup
       pool_config = config.extract!(:pool_size, :pool_timeout)
-      @pool = ConnectionPool.new(size: pool_config[:pool_size] || 5, timeout: pool_config[:pool_timeout] || 5 ) { Redis.new(config) }
+      @pool = ConnectionPool.new(size: pool_config[:pool_size] || 5, timeout: pool_config[:pool_timeout] || 5 ) do
```

Comment:
I quickly did that based on your message bus
We should probably have another method like: `Redis.new_with_namespace` and use it only. Or override Redis#initialize method (but I do not like that)

Then refactor the messagebus and the RedisPool

Reply:
That would be refactoring and can be done later. But I do agree.